MANIFEST.in

```diff
@@ -1,2 +0,0 @@
-include versioneer.py
```

Comment:
This file was autogenerated by `versioneer` command and contains list of files for source distribution:
https://docs.python.org/2/distutils/sourcedist.html

We should discuss whether this file is needed or not.